### PR TITLE
Devise Configuration

### PR DESCRIPTION
**Pull request by tayyabali12345**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,87 @@
...
+# Configure Using Gemfile
+# gem 'pg'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
+  host: localhost
+  username: postgres
+  password: root
```
Comment (devsinc) [inline config/database.yml]: should be managed through environment variables.

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (devsinc) [inline spec/models/user_spec.rb]: Remove these files

- Either we'll cater test cases fully
- Else we should remove these auto generated files

**Review by devsinc — CHANGES_REQUESTED** [review]

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  create_table "posts", force: :cascade do |t|
+    t.string "title"
+    t.text "body"
+    t.string "author"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "model_id"
+    t.index ["model_id"], name: "index_posts_on_model_id"
+  end
```
Comment (tiawana) [inline db/schema.rb:48]: there is no migrations, models related to these schema?

File: config/database.yml (hunk 1 above)
Comment (tayyabali12345) [reply config/database.yml]: Done

File: spec/models/user_spec.rb (hunk 2 above)
Comment (tayyabali12345) [reply spec/models/user_spec.rb]: Done

File: db/schema.rb (hunk 3 above)
Comment (tayyabali12345) [reply db/schema.rb:48]: Done, it is done in the Controller PR

File: .rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  TargetRailsVersion: 5.2
+  Exclude:
+    - "db/schema.rb"
+    - "node_modules/**/*"
+    - "vendor/bundle/**/*"
+    - "db/migrate/*"
+    - "bin/**/*"
+
+Layout/LineLength:
+  Max: 120
```
Comment (tiawana) [inline .rubocop.yml]: change it to 80

Comment (tayyabali12345) [reply .rubocop.yml]: done

**Pull request merged**